### ISSUE TITLE
Defect 5 - Critical - Skip to Main Content

### ISSUE THREAD

**Issue by srancour** [department-of-veterans-affairs/va.gov-cms#19375]
## Description or Additional Context
9/16/2024: The skip to main content link should take the user to main content of the page which is usually the H1 tag, but can be some other place in the content. The Skip to Main Content link always goes to the Manage link which is the first element in the block of content you should be skipping.

## Example
The "Skip to main content" on every page seems to always set the url to `#undefined` instead of the `#main-content` it's actually set to so the focus always goes to the next element instead of the main content on the page. ![Screenshot of upper portion of chrome with the url showing #undefined at the end and the Skip to main content link is visible and selected](https://github.com/user-attachments/assets/4fc6fc56-9ba3-49ad-8865-2b945150aa23)

### Code snippet
Skip to main content link `<a href="#main-content" class="visually-hidden focusable">Skip to main content</a>`
Main content tag `<div class="visually-hidden"><a id="main-content" tabindex="-1"></a></div>`

### Steps to reproduce
Navigate: Home or any other page
Tab on load to show the "Skip to main content" link, press link, tab to next element and focus moves to the "Manage" menu element.

## Recommended resolution
This seems to be a similar issue to #19368 and fixing one could fix both.

## Defect Criterion
[WCAG 2.4.1 - Bypass Blocks](https://www.w3.org/TR/WCAG21/#bypass-blocks) - A [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) is available to bypass blocks of content that are repeated on multiple [Web pages](https://www.w3.org/TR/WCAG21/#dfn-web-page-s).

### Users affected
- Keyboard only
- Screen reader

## Teams that this affects:
- [x] CMS Team

**Comment by srancour** [comment]
This also seems to have to be related with work that the last team had been doing as well in #12145.
